extensions/azurePublish/src/node/provision.ts

```diff
@@ -270,7 +271,9 @@ export class BotProjectProvision {
               displayName: config.hostname, // todo: this may be wrong!
               endpoint: `https://${provisionResults.webApp.hostname}/api/messages`,
               appId: provisionResults.appId,
+              webAppHostname: provisionResults.webApp.hostname
             });
+            provisionResults.bot = config.hostname;
```

Comment:
Why we added a bot field here? What is this used for, what's the difference with hostname?

Reply:
Let's keep this bot field for now, I will follow up with you offline. @zidaneymar @VanyLaw